src/ApiService/ApiService/onefuzzlib/Network.cs

```diff
@@ -50,7 +44,7 @@ public static async Async.Task<Network> Create(string region, ICreds creds, ICon
             }
             else
             {
-                var networkId = GenerateGuidv5(NETWORK_GUID_NAMESPACE, string.Join("|", networkConfig.AddressSpace, networkConfig.Subnet));
+                var networkId = Faithlife.Utility.GuidUtility.Create(NETWORK_GUID_NAMESPACE, string.Join("|", networkConfig.AddressSpace, networkConfig.Subnet), 5);
```

Comment:
we need to take a 3rd party dependency to generate network id ?

Reply:
Yes, There is not way generate a deterministic Guid with the base library. 
If you have a better suggestion let me know